src/drt/src/db/obj/frInstTerm.h

```diff
@@ -73,36 +73,16 @@ class frInstTerm : public frBlockObject
   bool hasAccessPoint(frCoord x, frCoord y, frLayerNum lNum);
   void getShapes(std::vector<frRect>& outShapes, bool updatedTransform = false);
   Rect getBBox();
+  void setOrderId(int in) { order_id_ = in; }
+  int getOrderId() const { return order_id_; }
 
  private:
+  int order_id_;
```

Comment:
I see many classes now have an "order_id_".  What is its purpose?

Reply:
it's the index of the object in the owner's table.

Reply:
Why is it now required?

Reply:
It's required in serialization as I only serialize this id and de-reference it at the receiving side. check serialization.h serializeBlockObject